coralproject/talk
# Adds a screen to create username after facebook signup

## What does this PR do?

- Adds a dialog window after singup to create/modify display Name.

## How do I test this PR?

- Signup with facebook
- Check that the dialog is there
- Change your display name
- Check that it was changed.

Story: https://www.pivotaltracker.com/story/show/134624635
